projects/storefrontstyles/scss/components/cart/_applied-coupons.scss

```diff
@@ -39,7 +39,7 @@
     }
 
     @include media-breakpoint-up(md) {
-      padding-inline-end: 0;
+      // padding-inline-end: 0;
```

Comment:
I think we can safely remove this entire media query 😉  https://github.com/SAP/spartacus/pull/14787#discussion_r789369831